spec/models/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+require "./app/models/user"
```

Comment:
Rails helper loads the Rails environment, thus line 2 is not needed.

If you weren't calling on `rails_helper`, then it would have been needed.

Reply:
Thanks Elle, ignore in all the next PRs as I will fix this across them all .